src/app/screens/AccountsList/styles.module.scss

```
@@ -0,0 +1,81 @@
+@import 'scss/variables/_sizes.scss';
+@import 'scss/variables/_fontSizes.scss';
+@import '_general.scss';
+
+$primary-color: #182C49;
+$secondary-color: #677489;
+$hover-color: hsla(217, 14%, 47%, 0.15);
+$shadow-color: rgba(0, 0, 0, 0.2);
+$gold-color: rgb(255, 217, 0);
+$gradient1-color: rgba(224, 99, 99, 1);
+$gradient2-color: rgba(90, 129, 226, 1);
+$gradient3-color: rgba(43, 191, 221, 1);
+
+$item-height: 250px;
+
+.container {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  height: calc(100% - #{$topbar-heigth});
+  justify-content: space-evenly;
+  padding: 40px 20px;
+  width: 100%;
+}
+
+.title {
+  font-size: #{$large};
+  padding: 40px 20px 0;
+  width: 100%;
+  text-align: center;
```

fijate el orden alfabético de las props! 

```suggestion
  font-size: #{$large};
  padding: 40px 20px 0;
  text-align: center;
  width: 100%;
```

---

Perfecto, corregido